lib/sfrest/centralized_role_management.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SFRest
+  # We disable that because we need to keep this naming due to connection.rb way of autoloading stuff and rubocop
+  # doesn't like it.
+  # rubocop: disable Naming/ClassAndModuleCamelCase
+  # Find Staging envs and stage a set of sites
+  class Centralized_role_management
+    # rubocop: enable Naming/ClassAndModuleCamelCase
+
+    # @param [SFRest::Connection] conn
+    def initialize(conn)
+      @conn = conn
+    end
+
+    # Get current centralized role management settings
+    #
+    # @return [Array] an array of roles with their associated settings.
+    def centralized_role_mapping
+      @conn.get('/api/v1/centralized-role-management')
+    end
+
+    # Set the centralized role management settings for a given role.
+    # @param [Boolean] enable the enablement of the mapping.
+    # @param [string] factory_role the site factory role.
```

Comment:
_Technically_ all the type names are capitalized (and classes)
```suggestion
    # @param [String] factory_role the site factory role.
```
But the codebase is already inconsistent here so /shrug

Reply:
Fixing :D